### DISCUSSION TITLE
Error FileNotFoundException,

### DISCUSSION THREAD

**Discussion by Elnur122** [qwbarch/mirage#124] [Q&A] [answered]
When I launch the game, an error appears. I use Manual Download and I move the files Mirage.dll, Mirage.Core.dll, Mirage.Compatibility.dll to the Beplnex/plugins folder
[Error  : Unity Log] Couldn't extract exception string from exception of type FileNotFoundException (another exception of class 'FileNotFoundException' was thrown while processing the stack trace)
[Error  : Unity Log]
Stack trace:

[Error  : Unity Log] FileNotFoundException: Could not load file or assembly 'FSharp.Core, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.
Stack trace:
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.Rendering.HighDefinition.HDRenderPipelineAsset:OnEnable()

**Comment by qwbarch** [comment, marked as answer]
You're missing dependencies. I highly recommend using a mod manager like ``r2modman`` instead so you don't run into dependency issues
